{{cookiecutter.repo_name}}/AUTHORS.md

```diff
@@ -0,0 +1,21 @@
+# Authors
+
+{{cookiecutter.project_name}} was created by {{cookiecutter.author_name}} in {% now 'utc', '%Y' %}.
+
+
+All contributing authors are listed in this file below.
+The repository history at https://github.com/{{cookiecutter.github_url}}
+and the CHANGELOG show individual code contributions.
+
+## Chronological list of authors
+
+<!--
+The rules for this file:
+  * Authors are sorted chronologically, earliest to latest
+  * Please give the name you go by, not your GitHub username
```

Comment:
In principle people should be allowed to name themselves anyway they like so I would write this more as a suggestion than a must. 

I also wish that we had started adding names + GitHub IDs as this would make it un-ambiguous and everyone easy to look up.

```markdown
**2022**
- A. U. Thor <the_author>
```

Reply:
Changed to:

```
<!--
The rules for this file:
  * Authors are sorted chronologically, earliest to latest
  * Please format it each entry as "Preferred name <GitHub username>"
  * Your preferred name is whatever you wish to go by --
    it does *not* have to be your legal name!
  * Please start a new section for each new year
  * Don't ever delete anything
-->

**{% now 'utc', '%Y' %}**
- {{cookiecutter.author_name}} \<@{{cookiecutter.github_username}}\>
- ```

Looks like (https://github.com/lilyminium/test_mda_condaforge_rtd/blob/main/AUTHORS.md):

**2022**

    cookie monster <@lilyminium>